### PR TITLE
Added test case to generate rke config

### PR DESCRIPTION
**Pull request by soumyalj**
Added a config file with 9 nodes

### CODE REVIEW

File: tests/validation/tests/v3_api/test_import_cluster.py [hunk 1]
```diff
@@ -65,15 +78,21 @@ def create_nodes():
...
 def create_rke_cluster_config(aws_nodes):
 
-    configfile = "cluster.yml"
+    # Create RKE Config file
+    print("Number of nodes ")
+    print(AWS_NODE_COUNT)
+    if (AWS_NODE_COUNT == 3):
+        configfile = "cluster.yml"
+    if (AWS_NODE_COUNT == 9):
+        configfile = "cluster_config_1.yml"
```
Comment (sangeethah) [inline tests/validation/tests/v3_api/test_import_cluster.py]: We should be able to generate the config file for any number of worker nodes without having to have any additional config yml. 
Existing config yml can be used for first 3 nodes and for the rest of the nodes build the configuration dynamically without relying on config yml.

Comment (soumyalj) [reply tests/validation/tests/v3_api/test_import_cluster.py]: Done

File: tests/validation/tests/v3_api/test_import_cluster.py [hunk 2]
```diff
@@ -6,34 +6,44 @@
...
     client = get_user_client()
 
-    # Create nodes in AWS
-    aws_nodes = create_nodes()
+    # Create AWS nodes for the cluster
+    aws_nodes = \
+        AmazonWebServices().create_multiple_nodes(
+            AWS_NODE_COUNT, random_test_name("rketestcustom"),
+            wait_for_ready=True)
+    assert len(aws_nodes) == AWS_NODE_COUNT
```
Comment (sangeethah) [inline tests/validation/tests/v3_api/test_import_cluster.py]: We should not rely on AWS_NODE_COUNT since we can only support provisioning up to 3 nodes .

Comment (soumyalj) [reply tests/validation/tests/v3_api/test_import_cluster.py]: We can rely on AWS_NODE_COUNT and changes will be made to support it.

**Pull request merged**